Makefile

```diff
@@ -20,10 +20,11 @@ SHELL := /bin/bash
 ARTIFACTS_BUCKET?=my-s3-bucket
 GIT_VERSION?=$(shell git describe --tag)
 GIT_TAG?=$(shell git tag -l --sort -v:refname | head -1)
-GOLANG_VERSION?="1.16"
-GO ?= $(shell source ./scripts/common.sh && build::common::get_go_path $(GOLANG_VERSION))/go
+GOLANG_VERSION?="1.17"
+GO_VERSION ?= $(shell source ./scripts/common.sh && build::common::get_go_path $(GOLANG_VERSION))
+GO ?= $(GO_VERSION)/go
 GO_TEST ?= $(GO) test
-
+export PATH := $(GO_VERSION):$(PATH)
```

Comment:
Where is go getting called that this is necessary? somewhere in one of the scripts this makefile is calling?

Reply:
One thing I observed was that the go1.17 was not in the path and because of this go1.17 was failing hence the need to add it to the path.

Reply:
I removed this in my commit, we can put it back if we need it, but since we are calling go via the makefile var im not sure why would need this...